tests/regressions/OverlayHeaderHeight-cy.js

```diff
@@ -1,45 +0,0 @@
-describe('Overlay Header Height [00j]', function () {
```

Comment:
I've adjusted the respective commit, to not delete this file, as we will reintroduce the header bar.